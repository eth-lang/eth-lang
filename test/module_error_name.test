module "Not an ident!" where
