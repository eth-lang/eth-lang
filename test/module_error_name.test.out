test/module_error_name.test:1:7 Expected module name, got "Not an ident!" of type 'string'
